[14/Jun/2021 10:45:05] INFO [django.utils.autoreload:636] Watching for file changes with StatReloader
[14/Jun/2021 10:45:05] INFO [django.utils.autoreload:636] Watching for file changes with StatReloader
[14/Jun/2021 10:55:30] INFO [django.utils.autoreload:636] Watching for file changes with StatReloader
[14/Jun/2021 10:55:30] INFO [django.utils.autoreload:636] Watching for file changes with StatReloader
[14/Jun/2021 10:55:32] WARNING [django.request:224] Not Found: /
[14/Jun/2021 10:55:32] WARNING [django.request:224] Not Found: /
[14/Jun/2021 10:55:32] WARNING [django.server:161] "GET / HTTP/1.1" 404 2486
[14/Jun/2021 10:55:32] WARNING [django.server:161] "GET / HTTP/1.1" 404 2486
[14/Jun/2021 10:55:35] INFO [django.server:161] "GET /omok/ HTTP/1.1" 200 5487
[14/Jun/2021 10:55:35] INFO [django.server:161] "GET /omok/ HTTP/1.1" 200 5487
[14/Jun/2021 10:55:37] INFO [django.server:161] "GET /omok/rooms/ HTTP/1.1" 200 9192
[14/Jun/2021 10:55:37] INFO [django.server:161] "GET /omok/rooms/ HTTP/1.1" 200 9192
[14/Jun/2021 10:55:46] INFO [django.server:161] "GET /omok/user/ HTTP/1.1" 200 10343
[14/Jun/2021 10:55:46] INFO [django.server:161] "GET /omok/user/ HTTP/1.1" 200 10343
[14/Jun/2021 10:56:40] INFO [django.utils.autoreload:250] /workspace/mysite/settings.py changed, reloading.
[14/Jun/2021 10:56:40] INFO [django.utils.autoreload:250] /workspace/mysite/settings.py changed, reloading.
[14/Jun/2021 10:57:53] INFO [django.utils.autoreload:636] Watching for file changes with StatReloader
[14/Jun/2021 10:57:53] INFO [django.utils.autoreload:636] Watching for file changes with StatReloader
[14/Jun/2021 10:57:55] WARNING [django.request:224] Not Found: /
[14/Jun/2021 10:57:55] WARNING [django.request:224] Not Found: /
[14/Jun/2021 10:57:55] WARNING [django.server:161] "GET / HTTP/1.1" 404 2486
[14/Jun/2021 10:57:55] WARNING [django.server:161] "GET / HTTP/1.1" 404 2486
[14/Jun/2021 10:57:57] INFO [django.server:161] "GET /omok/ HTTP/1.1" 200 5487
[14/Jun/2021 10:57:57] INFO [django.server:161] "GET /omok/ HTTP/1.1" 200 5487
[14/Jun/2021 10:58:00] ERROR [django.request:224] Internal Server Error: /omok/session/
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/django/core/handlers/exception.py", line 47, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.9/site-packages/django/core/handlers/base.py", line 181, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/usr/local/lib/python3.9/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/workspace/omok/session.py", line 104, in session
    body = json.loads(request.body)
  File "/usr/local/lib/python3.9/json/__init__.py", line 346, in loads
    return _default_decoder.decode(s)
  File "/usr/local/lib/python3.9/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/local/lib/python3.9/json/decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
[14/Jun/2021 10:58:00] ERROR [django.request:224] Internal Server Error: /omok/session/
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/django/core/handlers/exception.py", line 47, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.9/site-packages/django/core/handlers/base.py", line 181, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/usr/local/lib/python3.9/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/workspace/omok/session.py", line 104, in session
    body = json.loads(request.body)
  File "/usr/local/lib/python3.9/json/__init__.py", line 346, in loads
    return _default_decoder.decode(s)
  File "/usr/local/lib/python3.9/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/local/lib/python3.9/json/decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
[14/Jun/2021 10:58:01] ERROR [django.server:161] "GET /omok/session/ HTTP/1.1" 500 78283
[14/Jun/2021 10:58:01] ERROR [django.server:161] "GET /omok/session/ HTTP/1.1" 500 78283
[14/Jun/2021 10:58:41] ERROR [django.request:224] Internal Server Error: /omok/session/
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/django/core/handlers/exception.py", line 47, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.9/site-packages/django/core/handlers/base.py", line 181, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/usr/local/lib/python3.9/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/workspace/omok/session.py", line 104, in session
    body = json.loads(request.body)
  File "/usr/local/lib/python3.9/json/__init__.py", line 346, in loads
    return _default_decoder.decode(s)
  File "/usr/local/lib/python3.9/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/local/lib/python3.9/json/decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
[14/Jun/2021 10:58:41] ERROR [django.request:224] Internal Server Error: /omok/session/
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/django/core/handlers/exception.py", line 47, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.9/site-packages/django/core/handlers/base.py", line 181, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/usr/local/lib/python3.9/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/workspace/omok/session.py", line 104, in session
    body = json.loads(request.body)
  File "/usr/local/lib/python3.9/json/__init__.py", line 346, in loads
    return _default_decoder.decode(s)
  File "/usr/local/lib/python3.9/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/local/lib/python3.9/json/decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
[14/Jun/2021 10:58:41] ERROR [django.server:161] "GET /omok/session/ HTTP/1.1" 500 78199
[14/Jun/2021 10:58:41] ERROR [django.server:161] "GET /omok/session/ HTTP/1.1" 500 78199
[14/Jun/2021 10:58:44] INFO [django.server:161] "GET /omok/ HTTP/1.1" 200 5487
[14/Jun/2021 10:58:44] INFO [django.server:161] "GET /omok/ HTTP/1.1" 200 5487
[14/Jun/2021 10:58:46] WARNING [django.request:224] Forbidden: /omok/rooms/
[14/Jun/2021 10:58:46] WARNING [django.request:224] Forbidden: /omok/rooms/
[14/Jun/2021 10:58:46] WARNING [django.server:161] "GET /omok/rooms/ HTTP/1.1" 403 5130
[14/Jun/2021 10:58:46] WARNING [django.server:161] "GET /omok/rooms/ HTTP/1.1" 403 5130
[14/Jun/2021 11:01:44] INFO [django.utils.autoreload:250] /workspace/mysite/settings.py changed, reloading.
[14/Jun/2021 11:01:44] INFO [django.utils.autoreload:250] /workspace/mysite/settings.py changed, reloading.
[14/Jun/2021 11:01:45] INFO [django.utils.autoreload:636] Watching for file changes with StatReloader
[14/Jun/2021 11:01:45] INFO [django.utils.autoreload:636] Watching for file changes with StatReloader
[14/Jun/2021 11:01:47] WARNING [django.request:224] Not Found: /
[14/Jun/2021 11:01:47] WARNING [django.request:224] Not Found: /
[14/Jun/2021 11:01:47] WARNING [django.server:161] "GET / HTTP/1.1" 404 2486
[14/Jun/2021 11:01:47] WARNING [django.server:161] "GET / HTTP/1.1" 404 2486
[14/Jun/2021 11:01:50] INFO [django.server:161] "GET /omok/ HTTP/1.1" 200 5487
[14/Jun/2021 11:01:50] INFO [django.server:161] "GET /omok/ HTTP/1.1" 200 5487
[14/Jun/2021 11:01:56] WARNING [django.request:224] Forbidden: /omok/rooms/
[14/Jun/2021 11:01:56] WARNING [django.request:224] Forbidden: /omok/rooms/
[14/Jun/2021 11:01:56] WARNING [django.server:161] "GET /omok/rooms/ HTTP/1.1" 403 5130
[14/Jun/2021 11:01:56] WARNING [django.server:161] "GET /omok/rooms/ HTTP/1.1" 403 5130
[14/Jun/2021 11:02:11] INFO [django.utils.autoreload:250] /workspace/mysite/settings.py changed, reloading.
[14/Jun/2021 11:02:11] INFO [django.utils.autoreload:250] /workspace/mysite/settings.py changed, reloading.
[14/Jun/2021 11:02:11] INFO [django.utils.autoreload:636] Watching for file changes with StatReloader
[14/Jun/2021 11:02:11] INFO [django.utils.autoreload:636] Watching for file changes with StatReloader
[14/Jun/2021 12:07:24] INFO [django.utils.autoreload:636] Watching for file changes with StatReloader
[14/Jun/2021 12:07:24] INFO [django.utils.autoreload:636] Watching for file changes with StatReloader
[14/Jun/2021 12:07:27] WARNING [django.request:224] Not Found: /
[14/Jun/2021 12:07:27] WARNING [django.request:224] Not Found: /
[14/Jun/2021 12:07:27] WARNING [django.server:161] "GET / HTTP/1.1" 404 2486
[14/Jun/2021 12:07:27] WARNING [django.server:161] "GET / HTTP/1.1" 404 2486
[14/Jun/2021 12:07:31] INFO [django.server:161] "GET /omok/ HTTP/1.1" 200 5487
[14/Jun/2021 12:07:31] INFO [django.server:161] "GET /omok/ HTTP/1.1" 200 5487
[14/Jun/2021 12:07:56] INFO [django.utils.autoreload:250] /workspace/mysite/settings.py changed, reloading.
[14/Jun/2021 12:07:56] INFO [django.utils.autoreload:250] /workspace/mysite/settings.py changed, reloading.
[14/Jun/2021 12:07:57] INFO [django.utils.autoreload:636] Watching for file changes with StatReloader
[14/Jun/2021 12:07:59] WARNING [django.request:224] Not Found: /
[14/Jun/2021 12:07:59] WARNING [django.server:161] "GET / HTTP/1.1" 404 2486
[14/Jun/2021 12:08:04] INFO [django.server:161] "GET /omok/ HTTP/1.1" 200 5487
[14/Jun/2021 12:08:16] INFO [django.utils.autoreload:250] /workspace/mysite/settings.py changed, reloading.
[14/Jun/2021 12:09:03] INFO [django.utils.autoreload:636] Watching for file changes with StatReloader
[14/Jun/2021 12:09:08] INFO [django.utils.autoreload:250] /workspace/mysite/settings.py changed, reloading.
[14/Jun/2021 12:09:09] INFO [django.utils.autoreload:636] Watching for file changes with StatReloader
[14/Jun/2021 12:09:10] WARNING [django.request:224] Not Found: /
[14/Jun/2021 12:09:10] WARNING [django.server:161] "GET / HTTP/1.1" 404 2486
[14/Jun/2021 12:15:26] INFO [django.utils.autoreload:636] Watching for file changes with StatReloader
[14/Jun/2021 12:15:29] WARNING [django.request:224] Not Found: /
[14/Jun/2021 12:15:29] WARNING [django.server:161] "GET / HTTP/1.1" 404 2486
[14/Jun/2021 12:15:31] INFO [django.server:161] "GET /omok/ HTTP/1.1" 200 5487
[14/Jun/2021 12:15:36] WARNING [django.request:224] Forbidden: /omok/rooms/
[14/Jun/2021 12:15:36] WARNING [django.server:161] "GET /omok/rooms/ HTTP/1.1" 403 5130
[14/Jun/2021 12:15:41] WARNING [django.request:224] Forbidden: /omok/rooms/1/
[14/Jun/2021 12:15:41] WARNING [django.server:161] "GET /omok/rooms/1/ HTTP/1.1" 403 5285
[14/Jun/2021 12:16:53] INFO [django.utils.autoreload:250] /workspace/mysite/settings.py changed, reloading.
[14/Jun/2021 12:16:54] INFO [django.utils.autoreload:636] Watching for file changes with StatReloader
[14/Jun/2021 12:16:56] WARNING [django.request:224] Not Found: /
[14/Jun/2021 12:16:56] WARNING [django.server:161] "GET / HTTP/1.1" 404 2486
[14/Jun/2021 12:17:00] INFO [django.server:161] "GET /omok/ HTTP/1.1" 200 5487
[14/Jun/2021 12:17:06] WARNING [django.request:224] Not Found: /omok/1/
[14/Jun/2021 12:17:06] WARNING [django.server:161] "GET /omok/1/ HTTP/1.1" 404 4585
